django/contrib/gis/geos/prototypes/io.py

```diff
@@ -178,17 +179,27 @@ def read(self, wkb):
             raise TypeError
 
 
+def default_trim_value():
+    """
+    GEOS changed the default value in 3.12.0
+    can be replaced by a static True when 3.12.0 becomes the minimum supported version
+    """
+    return geos_version_tuple() >= (3, 12)
+
+
+DEFAULT_TRIM_VALUE = SimpleLazyObject(default_trim_value)
+
+
 # ### WKB/WKT Writer Classes ###
 class WKTWriter(IOBase):
     _constructor = wkt_writer_create
     ptr_type = WKT_WRITE_PTR
     destructor = wkt_writer_destroy
-
-    _trim = False
     _precision = None
 
     def __init__(self, dim=2, trim=False, precision=None):
         super().__init__()
+        self._trim = bool(DEFAULT_TRIM_VALUE)
```

Comment:
`bool()` is unnecessary
```suggestion
        self._trim = DEFAULT_TRIM_VALUE
```